src/index.html

```diff
@@ -11,7 +11,131 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="box">
+        <legend>
+          Personal information
+        </legend>
+        <label class="form-field">
+          Surname:
+          <input type="text" name = "surname">
+        </label>
+        <label class="form-field">
+          Name:
+          <input type="text" name = "name">
+        </label>
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            name = "age"
+            value = "12"
```

Comment:
```suggestion
          <input type="text" name="name">
        </label>
        <label class="form-field">
          How old are You?
          <input
            type="number"
            name="age"
            value="12"
```
Please fix all cases